### PR TITLE
Add self promotion document

### PR DESCRIPTION
**Pull request by vcarl**

### CODE REVIEW

**Review by thisRaptori — APPROVED** [review]
Looks good! Any idea where/how we're gonna link to these? Extra menu links vs submenu vs just linking people manually etc. Also wonder if there's any value in linking the titles directly to the channel in discord? 🤔

**Review by nickserv — APPROVED** [review]

**Comment by vcarl** [comment]
I was thinking about that @thisRaptori, I think right now the best we can do is in #welcome and channel headers, and sending it to people in response to posts we take down. In the future, if we opt to change how joining the server works, i think it'd be good to include there

**Comment by vcarl** [comment]
Also I dig linking to the channels from headers!

File: src/md-pages/promotion.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Self Promotion
+---
+
+Reactiflux's members do lots of cool stuff, and we'd love for you to share what you've done!
+
+We have 2 channels where self-promotion is expected. We ask that you share it only once, unless there's a significant milestone or it's relevant to someone's question.
+
+## #i-made-this
```
Comment (vcarl) [inline src/md-pages/promotion.md]:
```suggestion
## [#i-made-this](https://discord.gg/GaCSDfm)
```

File: src/md-pages/promotion.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+New side projects, libraries, or other works you want to show off. This is predominantly meant as a showcase for individuals.
+
+## #news-and-links
```
Comment (vcarl) [inline src/md-pages/promotion.md]:
```suggestion
## [#news-and-links](https://discord.gg/TEYnXKw)
```

**Pull request merged**